syntax = "proto3";

package org.simple.api.schema;
option java_multiple_files = true;
option java_package = "org.simple.api.schema";

message Greeting {
    string text = 1;
}

enum GreetResponseState {
    SUCCESS = 0;
    FAIL = 1;
}

message GreetResponse {
    GreetResponseState state = 1;
}

service GreetService {
    rpc sendGreeting(Greeting) returns (GreetResponse) {}
}
